specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/ServerAzureADAdministrators.json

```diff
@@ -0,0 +1,416 @@
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 ResourceNotFound - The requested resource was not found."
+          }
+        },
+        "x-ms-examples": {
+          "Gets a Azure Active Directory administrator.": {
+            "$ref": "./examples/AdministratorGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ServerAzureADAdministrators"
+        ],
+        "description": "Creates a new Server Active Directory Administrator or updates an existing server Active Directory Administrator.",
+        "operationId": "ServerAzureADAdministrators_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "administratorName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "ActiveDirectory"
+            ],
+            "x-ms-enum": {
+              "name": "AdministratorName",
+              "modelAsString": true
+            }
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The required parameters for creating or updating an Active Directory Administrator.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServerAzureADAdministrator"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated a Azure Active Directory administrator.",
+            "schema": {
+              "$ref": "#/definitions/ServerAzureADAdministrator"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 SubscriptionDoesNotHaveServer - The requested server was not found\n\n * 404 ServerNotInSubscriptionResourceGroup - Specified server does not exist in the specified resource group and subscription.\n\n * 404 OperationIdNotFound - The operation with Id does not exist.\n\n * 409 OperationCancelled - The operation has been cancelled by user.\n\n * 409 OperationInterrupted - The operation on the resource could not be completed because it was interrupted by another operation on the same resource.\n\n * 500 OperationTimedOut - The operation timed out and automatically rolled back. Please retry the operation."
+          },
+          "202": {
```

Comment:
Are both 202 and 201 used?

Reply:
202 is used, but 200/201 are returned from Location URI when the LRO is completed.